INSTALL.md

```diff
@@ -0,0 +1,81 @@
+# balena CLI Installation Instructions
```

Comment:
The installation instructions were combined and moved here. Previously they were partially in `capitanodoc.ts` (which was the content on the [docs web page](https://www.balena.io/docs/reference/cli/)) and partially in README.md. I have also rewritten or reformatted some bits here and there.

Reply:
@pdcastro I assume we will need an associated docs PR to start pulling this in right?